### PR TITLE
Minor upgrade of gerrit module

### PR DESCRIPTION
**Pull request by fejta**
/assign @amwat @BenTheElder 
/cc @krzyzacy 

The upgrade converted a string type to a Timestamp type, made the corresponding changes to our client.

### CODE REVIEW

File: prow/gerrit/adapter/adapter.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 			filters = append(filters, filter)
 		}
 		latestRev := change.Revisions[change.CurrentRevision]
-		created, err := time.Parse(layout, latestRev.Created)
+		created := latestRev.Created.Time
 		if err != nil {
```
Comment (amwat) [inline prow/gerrit/adapter/adapter.go]: We can remove this error handling now.

File: prow/gerrit/adapter/adapter_test.go [hunk 2]
```diff
@@ -31,7 +31,10 @@ import (
 	"k8s.io/test-infra/prow/config"
 )
 
-var timeNow = time.Date(1234, time.May, 15, 1, 2, 3, 4, time.UTC)
+var (
+	timeNow  = time.Date(1234, time.May, 15, 1, 2, 3, 4, time.UTC)
+	stampNow = gerrit.Timestamp{Time: timeNow}
```
Comment (amwat) [inline prow/gerrit/adapter/adapter_test.go]: nit: use `makeStamp`?

Comment (fejta) [reply prow/gerrit/adapter/adapter_test.go]: Done

File: prow/gerrit/adapter/adapter.go (hunk 1 above)
Comment (fejta) [reply prow/gerrit/adapter/adapter.go]: Done

**Pull request merged**